reports/src/main/java/com/ivy/reports/ReportScreenEvent.kt

```diff
@@ -1,15 +1,16 @@
 package com.ivy.reports
 
 import android.content.Context
-import com.ivy.data.transaction.TransactionOld
+import android.net.Uri
 
 sealed class ReportScreenEvent {
-    data class OnFilter(val filter: ReportFilter?) : ReportScreenEvent()
-    data class OnExport(val context: Context) : ReportScreenEvent()
-    data class OnPayOrGet(val transaction: TransactionOld) : ReportScreenEvent()
-    data class OnUpcomingExpanded(val upcomingExpanded: Boolean) : ReportScreenEvent()
-    data class OnOverdueExpanded(val overdueExpanded: Boolean) : ReportScreenEvent()
+    object Start : ReportScreenEvent()
+
     data class OnFilterOverlayVisible(val filterOverlayVisible: Boolean) : ReportScreenEvent()
```

Comment:
`data class OnFilterOverlayVisible(val filterOverlayVisible: Boolean) : ReportScreenEvent()`

This is confusing. What does it do? What's its purpose?